config/environments/development.rb

```diff
@@ -46,7 +46,7 @@
   # config.i18n.raise_on_missing_translations = true
 
   # Annotate rendered view with file names.
-  # config.action_view.annotate_rendered_view_with_filenames = true
+  config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
I left this line of code uncommented as part of the upgrade suggestion. The idea was to make it easier for developers to debug by adding comments in HTML to show which view template was responsible for rendering a specific section of a page.

If this becomes too noisy, we could comment this line of code!